miqa/core/rest/other_endpoints.py

```diff
@@ -27,7 +27,7 @@ class MIQAConfigView(APIView):
     def get(self, request):
         return Response(
             {
-                'artifact_options': [key for key in default_identified_artifacts()],
+                'artifact_options': list(default_identified_artifacts()),
```

Comment:
```suggestion
```

Let's remove this field, so that this is always read from the properties of the project.

Reply:
I ran into some weird issues implementing this on the Vue side, so I've reverted to using `default_identified_artifacts` for now. I'd love to come back to it at some point, but need to keep moving on lung's project.  Understand if you don't want to merge without this implemented.